docs/content/preview/releases/techadvisories/ta-23476.md

```diff
@@ -17,161 +17,105 @@ type: docs
-    ```
-
-    As expected, the row is present. However, most likely the internal row has a non-zero microsecond component, just that due to the YCQL API millisecond standard, it will always return `000` for the microsecond part.
-
-1. Select the row using WHERE condition as follows:
-
-    ```cql
-    SELECT * FROM test WHERE id = 1 AND ts = '2024-08-21 17:44:47.647000+0000';
-    ```
+With `cql_revert_to_partial_microsecond_support=true`
 
-    ```output
-     id | ts | v1 | v2
-    ----+----+----+----
-    ```
+```cql
 
-    ```cql
-    SELECT * FROM test WHERE id = 1 AND ts >= '2024-08-21 17:44:47.647000+0000' and ts < '2024-08-21 17:44:47.648000+0000';
-    ```
+desc test
+CREATE TABLE tempkeyspace.test (
+    id int PRIMARY KEY,
+    created_date timestamp
+) WITH default_time_to_live = 0
+    AND transactions = {'enabled': 'true'};
 
-    ```output
-      id | ts                              | v1 | v2
-     ----+---------------------------------+----+----
-       1 | 2024-08-21 17:44:47.647000+0000 |  5 | 10
-    ```
+INSERT INTO test(id, created_date) VALUES (1, currenttimestamp()) ;
 
-    A simple query with equality on the same (id and) timestamp value returned before does not match. However, changing this to a range query search for all timestamps within that millisecond does return the row, confirming that the row is there but the microsecond value is non-zero. Note that for the second query the upper bound is the next millisecond value (648 rather than 647) so it will match everything in [647000, 647999] range from the microsecond perspective.
+INSERT INTO test(id, created_date) VALUES (2, '2024-09-25 15:52:40.768819+0000') ;
 
-### Index scan case
+SELECT * FROM t1;
+ id | created_date
+----+---------------------------------
+  1 | 2024-09-30 05:54:12.340000+0000
+  2 | 2024-09-25 15:52:40.768000+0000
 
-1. Set up the keyspace and table as follows:
+SELECT * FROM test WHERE created_date='2024-09-30 05:54:12.340000+0000';
 
-    ```cql
-    CREATE KEYSPACE temp;
-    USE temp;
-    CREATE TABLE test(id int, ts timestamp, v1 int, v2 int, primary key( id, ts)) WITH transactions = {'enabled': 'true'};
-    CREATE INDEX ON test(v1);
-    ```
+ id | created_date
+----+--------------
 
-    This example includes the same keyspace and table as the preceding primary-key example, and adds an index.
-   (To continue the preceding example, just create the index and skip steps 1 and 2.)
+SELECT * FROM test WHERE created_date='2024-09-25 15:52:40.768000+0000';
 
-1. Insert a row using `currenttimestamp()` as follows:
+ id | created_date
+----+--------------
 
-    ```cql
-    INSERT INTO test(id, ts, v1, v2) VALUES (1, currenttimestamp(), 5, 10);
-    SELECT * FROM test;
-    ```
+```
 
-    ```output
-     id | ts                              | v1 | v2
-    ----+---------------------------------+----+----
-      1 | 2024-08-21 17:44:47.647000+0000 |  5 | 10
-    ```
+With `cql_revert_to_partial_microsecond_support=false`
```

Comment:
Make the examples as bullets. 
- With `cql_revert_to_partial_microsecond_support=false`

Reply:
Done